tests/mpi/distribute_mg_dofs_periodicity_01.mpirun=28.output

```diff
@@ -0,0 +1,7 @@
+
+DEAL::Number of cells: 1024
```

Comment:
That would certainly make it one of the longer running tests, but I think I'd be ok with it if you think that it adds something useful. On my 64 core machines, a testsuite run takes ~35 minutes. This might add 5-10 seconds to it, which is a lot for one test, but then we also don't add these kinds of tests very often.